tools/archetype/binding.test/src/main/resources/archetype-resources/src/test/java/__bindingIdCamelCase__HandlerTest.java

```diff
@@ -1,7 +1,12 @@
 #set( $dt = $package.getClass().forName("java.util.Date").newInstance() )
 #set( $year = $dt.getYear() + 1900 )
+#if( $vendorName == "Eclipse.org/SmartHome" )
+    #set( $copyright = "Contributors to the Eclipse Foundation" )
+#else
+    #set( $copyright = "by the respective copyright holders." )
+#end
 /**
- * Copyright (c) 2014,${year} Contributors to the Eclipse Foundation
+ * Copyright (c) 2014,${year} ${copyright}
```

Comment:
As we are anyhow on it: Would it be possible to add an optional property startYear and insert this here? If not set, it could default to 2014.

Reply:
sure, hold on.